lib/install.js

```diff
@@ -805,7 +805,35 @@ function showVersions(forks) {
     });
   });
 
-  versions = alphabetize(versions);
+  return {
+    versions: alphabetize(versions),
+    linkedVersions: linkedVersions
+  };
+}
+
+function inspectVersions(forks) {
```

Comment:
this `fork` argument actually isn't being used - not sure if I should be or not. A bit unclear on how this affects the output.